sandstone/apps/filebrowser/receivers.py

```diff
@@ -0,0 +1,12 @@
+from pydispatch import dispatcher
+# from posixfs import PosixFS
```

Comment:
This import breaks the code, as PosixFS uses the settings, which haven't loaded. Is there a better way to be able to use the PosixFS class? (I am putting the signal handling in the __init__.py file at the moment)